input/fsh/Instance-Variant.fsh

```diff
@@ -0,0 +1,32 @@
+Instance: variant-example-1
+InstanceOf: Observation
+Description: "Example of a fhir representation of a variant"
+Usage: #example
+* status = #final
+* category[0] = $observation-category#laboratory
+* code = $loinc#69548-6 "Genetic variant assessment"
+* valueCodeableConcept = $loinc#LA9633-4 "Present"
+* method = $loinc#LA26398-0 "Sequencing"
+* specimen = Reference(specimen-example-1)
```

Comment:
* Should there be a second reference to a `Task` i.e. that would provide provenance for the pipeline that identified this variant?

* Should there be a third reference to a `DocumentReference` i.e. a VCF, MAF  or gene panel report that includes this reference?

Reply:
That's a good point, but technically, it would probably best be suited to go into the discover report, which is basically a collection of variants. I didn't provide an example for that, here, and I don't think I actually incorporated the tasks and docrefs into that, so I'll need to figure out where those should go.

Reply:
This should now be taken care of, though, not from this particular instance. Have a look at the DiagnosticReport example to see how they now fit together.